packages/core/src/browser/style/sidepanel.css

```diff
@@ -285,18 +277,14 @@
 
 #theia-bottom-content-panel .p-TabBar-tab:not(.p-mod-current) {
     color: var(--theia-panelTitle-inactiveForeground);
-    border-top: var(--theia-border-width) solid  var(--theia-panel-background);
+    border-top: var(--theia-border-width) solid transparent;
```

Comment:
I am a bit confused, since using the intended border `--theia-tab-activeBorderTop`, re-adds the top border to tabs, even for the default `dark+` theme. However, it doesn't do that for VSCode. I'll look into this again later. In the meantime, I've added `--theia-tab-activeBorderTop` back.

Reply:
Ah, confusion lifted, our default `tab.activeBorderTop` color is quite outdated. VSCode simply leaves the color completely blank by default, which fixes the issue.